demos/pedestrian_tracker_demo/cpp/main.cpp

```diff
@@ -126,14 +143,46 @@ int main(int argc, char **argv) {
 
         bool should_save_det_log = !detlog_out.empty();
 
+        //model API part
+        std::vector<std::string> labels;
+        if (!FLAGS_labels.empty())
+            labels = DetectionModel::loadLabels(FLAGS_labels);
+
+        std::unique_ptr<ModelBase> detection_model;
+        if (FLAGS_at == "centernet") {
+            detection_model.reset(new ModelCenterNet(det_model, (float)FLAGS_t, labels));
```

Comment:
Since you are refactoring this demo, pls allign code style to camelCase

Reply:
I've done